### ISSUE TITLE
Failed to copy resources to /var/folders/qt/..

### ISSUE THREAD

**Issue by unexist** [volodya-lombrozo/hg-revision-plugin#3]
Just installed your plugin and this is from the error log:

```
[INFO] Hg project dir: /Users/unexist/Projects/quarkus-kind-mp-showcase
[INFO] Start scanning of hg project
Nov 25, 2020 8:21:58 PM com.aragost.javahg.log.JULLogger info
INFO: Copy resource /javahg.py to /var/folders/qt/ldm39hx14sv9_mrjqqbvmnmw0000gn/T/1606332118267-0/javahg.py
Nov 25, 2020 8:21:58 PM com.aragost.javahg.log.JULLogger error
SEVERE: stderr from Mercurial: *** failed to import extension javahg from /var/folders/qt/ldm39hx14sv9_mrjqqbvmnmw0000gn/T/1606332118267-0/javahg.py: module 'mercurial.cmdutil' has no attribute 'command'

INFO: Copy resource /styles/changesets.style to /var/folders/qt/ldm39hx14sv9_mrjqqbvmnmw0000gn/T/1606332118267-0/styles/changesets.style
Nov 25, 2020 8:21:58 PM com.aragost.javahg.log.JULLogger info
INFO: Copy resource /styles/changesets-eager.style to /var/folders/qt/ldm39hx14sv9_mrjqqbvmnmw0000gn/T/1606332118267-0/styles/changesets-eager.style
Nov 25, 2020 8:21:58 PM com.aragost.javahg.log.JULLogger info
INFO: Copy resource /styles/tags.style to /var/folders/qt/ldm39hx14sv9_mrjqqbvmnmw0000gn/T/1606332118267-0/styles/tags.style
```

Why does the plugin try to copy files to a non-existent folder, it probably never has the permissions to write to?

**Comment by volodya-lombrozo** [comment]
hg-revision-plugin based on javahg library (https://javahg.aragost.com/introduction.html). At least for now. Unfortunately, this library tries to include some extension for undefined purposes. We can't disable this behavior at this time.
We will try to get rid of the javahg library soon.

Thank you so much for your feedback.

**Comment by volodya-lombrozo** [comment]
de2e831b45b8f4bf1a7157ea0cd16d07f1e12458

**Comment by volodya-lombrozo** [comment]
This issue was resolved in version 0.10
